src/wesnoth.cpp

```diff
@@ -986,12 +986,16 @@ static std::string autodetect_game_data_dir(std::string exe_dir)
 	else if(filesystem::file_exists(exe_dir + "/../data/_main.cfg")) {
 		auto_dir = filesystem::normalize_path(exe_dir + "/..");
 	}
-	// In Windows debug builds, the EXE is placed away from the game data dir
-	// (in projectfiles\VCx\Debug), but the working directory is set to the
-	// game data dir. Thus, check if the working dir is the game data dir.
-	else if(filesystem::file_exists(filesystem::get_cwd() + "/data/_main.cfg")) {
-		auto_dir = filesystem::get_cwd();
```

Comment:
Looking for a `data` dir in the current working directory sounds like it could be useful on other platforms, not just Windows. So I think this should be left in, even if the newly-added `_WIN32`-only search is tried first.

Reply:
Hm this is true. I completely forgot that this had relevance outside of Windows (but also for non-standard Windows builds or installs) until much later.